scripts/core/ui/components/TopMenuDropdownButton.tsx

```diff
@@ -2,14 +2,14 @@
 
 import React from 'react';
 import classNames from 'classnames';
-import {gettext} from 'core/utils';
 
 export class TopMenuDropdownButton extends React.PureComponent<{
     onClick: () => void;
     disabled?: boolean;
     active: boolean;
     pulsate: boolean;
     'data-test-id'?: string;
+    title?: string;
```

Comment:
is it only for tooltip? If so, tooltip would be a clearer name that wouldn't get confused with button text